templates/mc_taco/templates.yaml

```diff
@@ -0,0 +1,86 @@
+dataset: mc_taco
+templates:
+  1b27afce-9748-44bd-9d82-9db4b815c292: !Template
+    id: 1b27afce-9748-44bd-9d82-9db4b815c292
+    jinja: 'Given the context,
+
+
+      {{sentence}}
+
+
+      observe the following QA pair and check if the answer is plausible:
+
+
+      Question: {{question}}
+
+
+      Answer: {{answer}} |||
+
+
+      {{["No", "Yes"][label]}}'
+    name: mc_taco_1
+    reference: Basic Context, QA Pair, ask for plausibility
+  38ab730f-1ed8-4362-99e1-c0d305aa056e: !Template
+    id: 38ab730f-1ed8-4362-99e1-c0d305aa056e
+    jinja: "I've been grappling with the temporal accuracy of this answer for a while:\n\
+      \nQ: \"{{question}}\"\n\nI have the following information: \"{{sentence}}\"\n\
+      \nA: \"{{answer}}\" \n\nThis is definitely not a/an ||| {{[\"plausible\", \"\
+      implausible\"]}} answer."
+    name: mc_taco_6
+    reference: Context, QA Pair, Negation
+  5bc98cb7-350e-471e-b986-ad52a46f403c: !Template
+    id: 5bc98cb7-350e-471e-b986-ad52a46f403c
+    jinja: 'There are five temporal categories: Event Duration, Event Ordering, Frequency,
+      Typical Time, Stationarity.
+
+
+      Out of the above temporal categories, which one does the question "{{question}}"
+      belong to? |||
+
+      {{["Event Duration", "Event Ordering", "Frequency", "Typical Time", "Stationarity"][category]}}'
+    name: mc_taco_4
+    reference: Temporal categories as part of the prompt
+  5e5cedef-b943-439a-a75a-1140478b0620: !Template
+    id: 5e5cedef-b943-439a-a75a-1140478b0620
+    jinja: 'I have the following passage:
+
+
+      {{sentence}}
+
+
+      My query is: "{{question}}"
+
+
+      I want an answer that makes "temporal sense". |||
+
+
+      {% if label %}
+
+      {{answer}}
+
+      {% else %}
+
+      I am not sure.
```

Comment:
If there is no answer, I would leave blank so that example is skipped. There's no reason to teach the model to say "I don't know" just because we don't have an annotation. Right now it produces both temporally plausible answers and says "I don't know" for the same input. For example:

```
Prompt + X

I have the following passage:

Durer's father died in 1502, and his mother died in 1513.

My query is: "How long was his mother ill?"

I want an answer that makes "temporal sense".

Y

6 months
```

and 

```
Prompt + X

I have the following passage:

Durer's father died in 1502, and his mother died in 1513.

My query is: "How long was his mother ill?"

I want an answer that makes "temporal sense".

Y

I am not sure.
```

Reply:
Makes sense, will fix.